save-cloud-common/src/commonMain/kotlin/org/cqfn/save/entities/ExecutionRequest.kt

```diff
@@ -12,6 +12,6 @@ import kotlinx.serialization.Serializable
 @Serializable
 data class ExecutionRequest(
     val project: Project,
-    val gitDto: GitDto,
+    val gitDto: GitDto?,
```

Comment:
When user create new project, he could ignore GitHub(specify it later or upload binary). I thought it is redundantly to create new same structure to save new project, but with nullable field.

Reply:
It's not a big deal to have slightly overlapping entities; when we add new fields in the future it will be easier, if we have separate classes. Also, it will be more readable, because having `ExecutionRequest` be sent for registration is very confusing. I suggest to add a new class